src/server/goog-device/mw/WebsocketProxyOverAdb.ts

```diff
@@ -196,8 +211,11 @@ export class WebsocketProxyOverAdb extends WebsocketProxy {
             })
             .catch((e) => {
                 const msg = `[${this.TAG}] Failed to start service: ${e.message}`;
-                console.error(Utils.getTimeISOString(), udid, msg);
                 ws.close(4005, msg);
+                console.error(Utils.getTimeISOString(), udid, msg);
+                if (!e.handled) {
+                    Utils.captureMessage(msg, 'Android', udid);
+                }
```

Comment:
@metocks // iOS쪽과 동일하게 수정해 보았습니다.
```suggestion
               const self = this;
               Sentry.captureException(e, scope => {
                   scope.setTags('ramiel_device_type', 'Android');
                   scope.setTags('ramiel_device_id', self.udid);
                   scope.setTags('ramiel_message', e.ramiel_message);
                   return scope;
               });
```